scripts/upgradeable/transpile.sh

```diff
@@ -39,6 +39,10 @@ npx @openzeppelin/upgrade-safe-transpiler -D \
   -x '!contracts/proxy/utils/UUPSUpgradeable.sol' \
   -x '!contracts/proxy/beacon/IBeacon.sol' \
   -p 'contracts/**/presets/**/*' \
```

Comment:
We should remove that line since we don't have any presets any more, right?
```suggestion
```

Reply:
Yeah, that's right!